### PR TITLE
feat(pubsublite): add seek subscription sample

### PR DESCRIPTION
**Pull request by tmdiep**
Adds a new sample for the `SeekSubscription` feature:
https://pkg.go.dev/cloud.google.com/go/pubsublite#AdminClient.SeekSubscription

### CODE REVIEW

**Comment by tmdiep** [comment]
There are some unrelated test failures.

File: pubsublite/admin/seek_subscription.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+// [START pubsublite_seek_subscription]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/pubsublite"
+)
+
+func seekSubscription(w io.Writer, projectID, region, zone, subID string, seekTarget pubsublite.SeekTarget, waitForOperation bool) error {
```
Comment (codyoss) [inline pubsublite/admin/seek_subscription.go:26]: nit: Please add a doc comment: https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md#comment-functions-and-packages

**Review by codyoss — APPROVED** [review]
LGTM, just one minor thing

File: pubsublite/admin/seek_subscription.go (hunk 1 above)
Comment (tmdiep) [reply pubsublite/admin/seek_subscription.go:27]: Done

**Comment by codyoss** [comment]
@tmdiep Sorry about that, this should now auto merge on green. The failing test has been skipped.

**Comment by tmdiep** [comment]
Thanks for quickly resolving, Cody!

**Pull request merged**